src/Makefile.am

```diff
@@ -122,7 +124,6 @@ nobase_include_HEADERS =                                        \
   google/protobuf/reflection.h                                  \
   google/protobuf/reflection_ops.h                              \
   google/protobuf/repeated_field.h                              \
-  google/protobuf/repeated_field_reflection.h                   \
```

Comment:
It's deleted internally, but the deletion is lost from internal to opensource integration. See http://cl/71942508